orkenstein/filmustage-dataapi
# Test danger.system (#2)

* Test danger.system

* Try danger without bundler

* Try add ruby image

* Ruby version

* Fix rub

* Try custom build image

* Test coverage warnings
Custom dotnet image update
Some ignores

* Danger cobertura plugin + bundle cache

* Indent fix

* Check env

* Update dockerfile

* Move bundle install to docker image

* Verbose danger output

* Trigger changes

* Run tests last and fail coverage

* Reorder again
Touch project

* TMP: fail on test coverage early
